app.py

```diff
@@ -0,0 +1,81 @@
+"""Application for managing tarantool repositories."""
+
+from argparse import ArgumentParser
+import json
+import os
+
+from flask import Flask
+
+from s3repo.model import S3AsyncModel
+from s3repo.view import S3View
+
+
+class HTTPServerApp:
+    """A wrapper for working with the Flask HTTP server."""
+    def __init__(self, cfg):
+        self.app = Flask(__name__)
+        self.cfg = cfg
+
+    def set_view(self, path, view_func, methods):
+        """Add view for HTTP server."""
+        self.app.add_url_rule(path, view_func=view_func, methods=methods)
+
+    def run(self):
+        """Run the HTTP server."""
+        port = self.cfg['port'] or 5000
+        self.app.run(port=port)
+
+
+def parse_cfg():
+    """Parse the config specified using command line arguments."""
+    parser = ArgumentParser()
+    parser.add_argument('-c', '--cfg', dest='config',
+        help='configuration file', metavar='FILE')
```

Comment:
I would support some meaningful default. My usual repository layout is more or less so:

```
.gitignore (which contains config.json)
config.json.example
my_script.py
README.md (which mentions that one should create config.json by copying and editing config.json.example).
requirements.txt
```

Reply:
Now the contents of the repository are very similar to the one presented.